config/routes.rb

```diff
@@ -26,6 +26,8 @@
   devise_for :users, :controllers => { :omniauth_callbacks => "callbacks" }
   concern :exportable, Blacklight::Routes::Exportable.new
 
+  get '/catalog/destroy/:id' => 'catalog#destroy', :as => :destory_solr_document
```

Comment:
`destory` -> destroy

Reply:
THAT'S WHY MY PATH NAME WAS NOT WORKING! Thank you!